src/containers/__tests__/index.test.js.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Atebits } from '../atebits.js';
```

Comment:
Hey all of these look like perfect snapshot tests, great work.

Reply:
Thanks! Happy to have made my first contribution to someone's project 😃